fe/lib/components/BrandSelect/BrandTile.tsx

```diff
@@ -0,0 +1,95 @@
+import { Box, Stack, Text } from 'braid-design-system';
+import React from 'react';
+
+import { CopyableOid } from '../../private/CopyableOid';
+import {
+  AdvertisementBrandingFieldsFragment,
+  AdvertisementBrandingImage,
+} from '../../types/seekApi.graphql';
+
+import * as styles from './styles.css';
+
+const GREY_PIXEL =
+  'data:image/gif;base64,R0lGODlhAQABAIAAAMLCwgAAACH5BAAAAAAALAAAAAABAAEAAAICRAEAOw==';
```

Comment:
1. Are there cases where we could get a brand without a logo?
2. Should this be unified with the chequerboard pattern used for cover images?

Reply:
Added a `No logo` case but realised that I cannot create a Brand without a logo in AdCentre, so just simplify no need  `No Logo` case

Reply:
Sorry for jumping on this late, but that’s correct. It’s also documented on the Developer Site:
> A mandatory logo displayed on the hirer’s job ads and on SEEK’s native apply form. This is represented as an AdvertisementBrandingImage  with a type code of OriginalLogo.